src/Hpp/RunHpp.hs

```diff
@@ -444,10 +455,10 @@ directive = lift (onElements (awaitJust "directive")) >>= aux
                         let tokStr = toChars (detokenize toks)
                         throwError $ UserError ln (tokStr++" ("++curFile++")")
           "warning" -> True <$ lift dropLine -- warnings not yet supported
-          -- t -> do toks <- lift takeLine
-          --         ln <- subtract 1 <$> use lineNum
-          --         throwError $ UnknownCommand ln (detokenize (Important t:toks))
-          _ -> return False -- Ignore unknown command
+          t -> do toks <- lift takeLine
```

Comment:
I think this strict handling of unknown command should be behind some flag since I think normal C preprocessors allow unknown commands by simply ignoring them. Practically, it helps us out when we get errors on this since we know when CPP was used inappropriately.